docs/recipes/babelrc.md

```diff
@@ -13,7 +13,9 @@ There are multiple options for configuring how AVA transpiles your tests using B
 
 ## AVA's default transpiler behavior
 
-By default, AVA transpiles your tests and helper files using the [`@ava/stage-4`](https://github.com/avajs/babel-preset-stage-4) Babel preset. This is a great option for small modules where you do not desire a build step to transpile your source before deploying to `npm`.
+AVA by default provides some nifty ECMAScript features, like [`async`](https://github.com/avajs/ava#async-function-support), to your test-environment. To make this possible AVA transpiles the tests and helper files using the [`@ava/stage-4`](https://github.com/avajs/babel-preset-stage-4) Babel preset.
```

Comment:
I am somehow torn between using JS over ECMAScript for these reasons:

- JavaScript sounds just slight and handy
- This paragraph is about the differences between good old vanilla JS and the new (not backward compatible) language features introduced by ES.

I think this topic is read by people who already know the term "ECMAScript" and i think in this case it supports understanding (and avoids confusion for less experienced programmers ["Isn't that feature an ES-feature?"]).

But i am really not insisting on that - so if you'd prefer using JS, i am fine with that.